csharp/ql/test/library-tests/csharp9/PrintAst.expected

```diff
@@ -0,0 +1,284 @@
+Record.cs:
+#    4| [NamespaceDeclaration] namespace ... { ... }
+#    6|   1: [Class] IsExternalInit
+#    9| [Class] Person
+#    9|   11: [Property] EqualityContract
+#    9|     3: [Getter] get_EqualityContract
+#    9|   11: [NEOperator] !=
+#-----|     2: (Parameters)
+#    9|       0: [Parameter] r1
+#    9|       1: [Parameter] r2
+#    9|   11: [EQOperator] ==
+#-----|     2: (Parameters)
+#    9|       0: [Parameter] r1
+#    9|       1: [Parameter] r2
```

Comment:
It's a bit strange that these are not marked with compiler generated attribute. Should we exclude them from the tree?